.travis.yml

```
@@ -6,6 +6,9 @@ node_js:
   - "8"
   - "6"
 
+before_install:
+  - npm install -g npm@latest
```

```suggestion
  - nvm install-latest-npm
```
this has much more granular logic than always installing the latest - unless you want it to fail when the latest version doesn’t work.